cmd/src/batch_common.go

```diff
@@ -270,6 +270,19 @@ func batchExecute(ctx context.Context, out *output.Output, svc *batches.Service,
 		Timeout:     flags.timeout,
 		TempDir:     flags.tempDir,
 		Parallelism: flags.parallelism,
+		FetchFileMatches: func(repo *graphql.Repository) (map[string]bool, error) {
```

Comment:
Can we make this a method on `svc` and pass that in instead, like `FetchFileMatches: svc.FetchFileMatches`?

(We've talked about this on Zoom, but I don't like how here all the layers bleed into each other: the command-layer passing things from the service into the executor, but it's also not _too bad_ as to be a blocker, IMHO, so just leaving this as a note)